script/libtest.bash

```diff
@@ -295,6 +301,10 @@ test_kubeless_ingress() {
     kubeless route delete ing-${func}
 }
 test_kubeless_autoscale() {
```

Comment:
it would be a bit clearer if you do this in the `test/integration-tests.bats`:
```
@test "Test function autoscale: get-python" {
  if kubectl api-versions | grep -q "autoscaling/v2beta1"; then
    skip "Autoscale is only supported for Kubernetes >= 1.8"
  fi
  test_kubeless_autoscale get-python
  kubeless_function_delete get-python
}
```

that way the test will be marked as "skipped" instead of "passing" if it has not been executed.